config/kafka-rest.properties

```diff
@@ -17,3 +17,4 @@
 #id=kafka-rest-test-server
 #schema.registry.url=http://localhost:8081
 #zookeeper.connect=localhost:2181
+#bootstrap.servers=localhost:9092
```

Comment:
It's worth noting here that this can be autogenerated by running the `main()` in `KafkaRestConfig`